### ISSUE TITLE
[Bug]: Left sidepane transparency on mobile web view should be less or not at all.

### ISSUE THREAD

**Issue by small1** [nextcloud/server#50861]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Left sidepane when you zoom in to much or go on mobile view has a transparency set when opened. With underlying objects it gets hard to read and see the actual menu.

See attached screenshot.

![Image](https://github.com/user-attachments/assets/56ef3093-9c2d-45f6-bc90-f39edf09b60f)

### Steps to reproduce

1. minimize view or zoom in
2. open left sidepane
3. See it hovering over the file view. 

### Expected behavior

Less transparent background or solid background

### Nextcloud Server version

30

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.3

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Fresh Nextcloud Server install

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [x] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```json

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell
none
```

### Nextcloud Logs

```json

```

### Additional info

This can be seen on at least  Nextcloud 30 and 31rc

**Comment by small1** [comment]
The same behavior is in nc31 rc as well.

**Comment by szaimen** [comment]
@nextcloud/designers is this something we should change?

**Comment by kra-mo** [comment]
> [@nextcloud/designers](https://github.com/orgs/nextcloud/teams/designers) is this something we should change?

If blur is not available in the browser, yes.

**Comment by kra-mo** [comment]
~~But I actually cannot reproduce this myself. Is this still happening with the latest version?~~

Oh, because I do have blur! Yeah, this should not happen when it isn't there. We should have a solid background.

**Comment by small1** [comment]
Any progress?

**Comment by jhnc** [comment]
I notice than on the same browser (ubuntu snap firefox on desktop), my normal nextcloud account has the problem but the admin account gets a solid background. So there seems to be some sort of hidden per-user (role?) setting for this.

update: it turns out that toggling background blur on my normal account does seem to fix the issue but only if I exit and log back in, which I had not thought to do before.

**Comment by jancborchardt** [comment]
Adding as papercut, FYI @sorbaugh @susnux

**Comment by artfulrobot** [comment]
This transparency is so annoying. It affects me without any zoom or mobile, just normal desktop.

I'm using LibreWolf/Firefox which does support blur, but the NC CSS seems to not use the feature: it gives me

```
  --filter-background-blur : none;
```

while setting an 0.8 opacity on the background colour in `--color-main-background-blur` which is unreadable.

I've installed Custom CSS in order to add this:

```
:root {
  --color-main-background-blur: white;
}
```

to get rid of the transparency.

If you really want the blur (because you value some questionable aesthetics over accessibility, for example) you can add this instead and keep your transparency.

```css
:root{
  --filter-background-blur : blur(8px);
}
```
